vendor/assets/javascripts/highlightjs/languages/brainfuck.min.js

```diff
@@ -1 +1,7 @@
-hljs.registerLanguage("brainfuck",function(){"use strict";return function(e){var n={className:"literal",begin:"[\\+\\-]",relevance:0};return{name:"Brainfuck",aliases:["bf"],contains:[e.COMMENT("[^\\[\\]\\.,\\+\\-<> \r\n]","[\\[\\]\\.,\\+\\-<> \r\n]",{returnEnd:!0,relevance:0}),{className:"title",begin:"[\\[\\]]",relevance:0},{className:"string",begin:"[\\.,]",relevance:0},{begin:/(?:\+\+|\-\-)/,contains:[n]},n]}}}());
\ No newline at end of file
+hljs.registerLanguage("brainfuck",(()=>{"use strict";return e=>{const n={
```

Comment:
Interesting that `.min.js` files are no longer oneliners. But not **that** interesting to see what changed in highlight.js build pipeline 😜